hail/src/test/scala/is/hail/TestUtilsSuite.scala

```diff
@@ -10,58 +10,6 @@ import org.testng.annotations.{DataProvider, Test}
 
 class TestUtilsSuite extends SparkSuite {
 
-  def getClasses(packageName: String): Array[Class[_]] = {
-    val classLoader = Thread.currentThread().getContextClassLoader()
-    assert(classLoader != null)
-    val path = packageName.replace(".", "/")
-    val resources = classLoader.getResources(path)
-    val dirs = new ArrayBuilder[File]()
-    while (resources.hasMoreElements) {
-      val resource = resources.nextElement()
-      val uri = new URI(resource.toString)
-      dirs += new File(uri.getPath())
-    }
-    val classes = new ArrayBuilder[Class[_]]()
-    for (directory <- dirs.result()) {
-      classes ++= findClasses(directory, packageName)
-    }
-    classes.result()
-  }
-
-  def findClasses(dir: File, packageName: String): Array[Class[_]] = {
-    val classes = new ArrayBuilder[Class[_]]()
-    if (!dir.exists())
-      return classes.result()
-
-    val files = dir.listFiles()
```

Comment:
Is this not needed anymore? I had put it in because the tests with data providers wouldn't fail if the data provider failed.

Reply:
No, it's broken, I created another issue: https://github.com/hail-is/hail/issues/6079.  The code didn't work for a test jar, but skipped test show up as failures so our test coverage isn't diminished (but readable error messages might be).